### ISSUE TITLE
module fails with test example

### ISSUE THREAD

**Issue by kostyrev** [vgeyer/puppet-hosts#1]
I've got 
CentOS release 6.5
puppet-server-3.6.2-1.el6.noarch
├── puppetlabs-stdlib (v4.1.0)

I included this in my node's definition;
`class { 'hosts':
    hosts => {
        'sidious.local' => {alt =>['sidious', 'puppet'], ipv4 => '192.168.2.1', ipv6 => 'fdc9:7739:4b55:da6:0:0:0:2'},
        'vader.local' => {alt =>['vader'], ipv4 => '192.168.2.2', ipv6 => 'fdc9:7739:4b55:da6:0:0:0:10'}
    }
}`

it gives me
```Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Failed to parse template hosts/etc/hosts.erb:
  Filepath: /usr/lib/ruby/site_ruby/1.8/puppet/parser/scope.rb
  Line: 798
  Detail: undefined method`function_validate_ipv4_address' for #Puppet::Parser::Scope:0x7f4a532edc78
 at /etc/puppet/modules/hosts/manifests/init.pp:25 on node puppet-client.local
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run

```
```

**Comment by vgeyer** [comment]
The Module deps are wrong. stdlib >=4.2.0 are required were the validate_ipv4_address function was added.

**Comment by vgeyer** [comment]
Fixed in 63684f3927f271aa4f0bef10b6ef32faf57af08d and deade4a02129b26542b25be1cc7dbfddacc38ff1

Pushed to forge repo.
